runtime/src/main/java/org/corfudb/runtime/collections/StreamManager.java

```diff
@@ -42,7 +42,7 @@
 public class StreamManager {
 
     private static final int DEFAULT_NUM_SUBSCRIBERS = 6; // Max number of threads.
-    private static final int MAX_SUBSCRIBERS = 32; // Arbitrary sanity check value.
+    public static final int MAX_SUBSCRIBERS = 32; // Arbitrary sanity check value.
```

Comment:
Why is this public?

Reply:
So we can access it in a test. There was no test for this edge case (when more than MAX_SUBSCRIBERS subscribe)..